### DISCUSSION TITLE
Vercel cron jobs do not run locally

### DISCUSSION THREAD

**Discussion by jakeleventhal** [vercel/community#2444] [Help] [answered]
I have cron jobs configured in my vercel.json, but when I run a local devserver, those jobs don't run

**Comment by leerob** [comment, marked as answer]
This is intended and expected: cron jobs only run in production. To test locally, you can manually hit the desired API endpoint the cron is pointing at 👍 

https://vercel.com/docs/cron-jobs

**Reply by jakeleventhal** [reply]
Why wouldn't we want to have an option in the vercel.json to opt in to running locally?
